app.js

```diff
@@ -0,0 +1,72 @@
+const Helper = require('./helper.js');
+const chalk = require('chalk');
+const yargs = require('yargs');
+
+const addOptions = {
+    command: "add",
+    describe: "add Note",  
+    
+   builder: {
+       body: {
+           describe: 'body',
+           type: 'string'
+       },
+
+       title: {
+           describe: 'body',
+           type: 'string',
+       }
+   },
```

Comment:
please fix indentation: on the same level as 
```
    command: "add",
    describe: "add Note",
```
remove empty lines before and after ``` builder: {...} ```